nix/fetchNixpkgs.nix

```diff
@@ -1,49 +1,8 @@
-{ rev                             # The Git revision of nixpkgs to fetch
```

Comment:
We should keep the rest of this file, actually. Many people have not upgraded to Nix 1.12 or 2.0 yet and this would break things for them.

Reply:
Note that your change to this file caused the circleci build to fail (which uses an older version of Nix).